# Evolution of the tyrosinase gene family in bivalve molluscs: Independent expansion of the mantle gene repertoire - Supplementary Data

This GitHub repository provides the supplementary data referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Evolution of the tyrosinase gene family in bivalve molluscs: Independent expansion of the mantle gene repertoire.** Felipe Aguilera, Carmel McDougall, Bernard M. Degnan. *Acta Biomaterialia* **10**:3855-3865 (2014). https://doi.org/10.1016/j.actbio.2014.03.031.

**Abstract**

Tyrosinase is a copper-containing enzyme that mediates the hydroxylation of monophenols and oxidation of o-diphenols to o-quinones. This enzyme is involved in a variety of biological processes, including pigment production, innate immunity, wound healing, and exoskeleton fabrication and hardening (e.g. arthropod skeleton and mollusc shell). Here we show that the tyrosinase gene family has undergone large expansions in pearl oysters (Pinctada spp.) and the Pacific oyster (Crassostrea gigas). Phylogenetic analysis reveals that pearl oysters possess at least four tyrosinase genes that are not present in the Pacific oyster. Likewise, C. gigas has multiple tyrosinase genes that are not orthologous to the Pinctada genes, indicating that this gene family has expanded independently in these bivalve lineages. Many of the tyrosinase genes in these bivalves are expressed at relatively high levels in the mantle, the organ responsible for shell fabrication. Detailed comparisons of tyrosinase gene expression in different regions of the mantle in two closely related pearl oysters, P. maxima and P. margaritifera, reveals that recently evolved orthologous tyrosinase genes can have markedly different expression profiles. The expansion of tyrosinase genes in these oysters and their co-option into the mantleâ€™s gene regulatory network is consistent with mollusc shell formation being underpinned by a rapidly evolving transcriptome.

## Author contact

- [Felipe Aguilera](mailto:f.aguilera@uq.edu.au) (first author - PhD student)
- [Carmel McDougall](mailto:c.mcdougall@uq.edu.au) (senior author - PostDoc)
- [Bernie Degnan](b.degnan@uq.edu.au) (corresponding author)

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary-Figures-Sequences.docx](https://github.com/faguil/Tyrosinase-Evolution-Molluscs/blob/main/01-Supplementary-Figures-Sequences.docx) contains supplementary figures and tyrosinase sequences used in this study.

- **Figure S1.** Neighbour-Joining (NJ) phylogenetic tree of molluscan tyrosinase proteins obtained by MEGA5.2.2 under the JTT + G substitution model. 
- **Figure S2.** Maximum Likelihood (ML) phylogenetic tree of molluscan tyrosinase proteins obtained by RAxMLGUI v1.3 under the PROTGAMMAWAG substitution model.
- **Figure S3.** Bayesian Inference (BI) phylogenetic tree of molluscan tyrosinase proteins obtained by MrBayes v3.2 under the WAG + G substitution model.
- **Figure S4.** Sequence alignments of hypothetical proteins that belong to the microsynteny block between *C. gigas* scaffold 867 and *P. fucata* scaffolds 13287, 1286 and 19072.
- **Figure S5.** Quantitative PCR expression profiles of tyrosinase genes across the mantle tissue in *P. maxima* and *P. margaritifera*.
- **Supplementary dataset file S1.** Tyrosinase protein sequences used in this study.
- **Supplementary dataset file S2.** List of *Pinctada margaritifera* reference gene sequences used for qPCR analysis.
